examples/stable_diffusion.py

```diff
@@ -384,15 +363,8 @@ def __call__(self, hidden_states, causal_attention_mask):
     src_len = key_states.shape[1]
     value_states = value_states.reshape(*proj_shape)
 
-    attn_weights = query_states @ key_states.permute(0,2,1)
-
-    attn_weights = attn_weights.reshape(bsz, self.num_heads, tgt_len, src_len) + causal_attention_mask
-    attn_weights = attn_weights.reshape(bsz * self.num_heads, tgt_len, src_len)
-
-    attn_weights = attn_weights.softmax()
-
-    attn_output = attn_weights @ value_states
-
+    causal_attention_mask = causal_attention_mask.reshape(bsz * self.num_heads, tgt_len, src_len)
```

Comment:
Removing this one line resolves the issue completely, here are the images that were generated with it removed. Same prompts.
![image](https://github.com/tinygrad/tinygrad/assets/19949489/f807fdc0-7f46-4497-95cf-7749b4df82ef)
![image](https://github.com/tinygrad/tinygrad/assets/19949489/189bdb5a-c14a-4b6e-bf61-8b44e8c33a73)